docs/CONFIG.md

```diff
@@ -0,0 +1,123 @@
+# Privacy Pass configuration
+
+Privacy Pass uses cryptographic tokens to bypass internet challenges from certain providers. In this doc, we discuss the configuration file that Privacy Pass uses for interpreting when tokens should be sent for signing/redemption. See config.js for an example configuration (exampleConfig) describing the correct format of the JSON struct.
+
+## config.js
+
+Holds the various configurations as JavaScript JSON structs for the providers that Privacy Pass interacts with. Currently, only the Cloudflare config is active. There is an example conifg, there is also an in progress config for the CAPTCHA provider FunCAPTCHA. In the following we will highlight how each config field is used.
+
+### config["id"]
+
+A unique identifier highlighting which config is used. Currently cfConfig.id = 1, all other configs should be added with unique "id" values.
+
+### config["sign"]
+
+A bool dictating whether Privacy Pass should send tokens for signing. 
+
+### config["redeem"]
+
+A bool dictating whether Privacy Pass should send tokens for redemption.
+
+### config["sign-reload"]
+
+A bool dictating whether the page should be reloaded after tokens are successfully signed
+
+### config["sign-resp-format"]
+
+Format of the response (as a string) to a signing request. Currently, support "string" or "json". When "string" is used, expect the signed tokens to be included (base-64 encoded) in the HTTP response body in the form `signatures= || <signed-tokens> || <Batch-DLEQ-Resp>`. When "json" is used, expect the signed tokens to be included (base-64 encoded) as a JSON struct with key: "signatures" and value `<signed-tokens> || <Batch-DLEQ-Resp>`.
+
+### config["storage-key-tokens"]
+
+The string key under which the tokens for the active config are stored in local storage. This should be unique for each config to prevent tokens being spent for inconsistent providers.
+
+### config["storage-key-count"]
+
+The string key corresponding to the number of tokens currently held in local storage (under config["storage-key-tokens"]). This should also be unique per config.
```

Comment:
"This should also be unique per config." 
Why?

Reply:
Removed. Using the same id method as the previous comment.